internal/action/sync.go

```diff
@@ -181,11 +204,12 @@ func (s *Action) syncMount(ctx context.Context, mp string) error {
 	}
 	syncPrintDiff(ctxno, l, ln)
 
-	debug.Log("Syncing Mount %s. Exportkeys: %t", mp, ctxutil.IsExportKeys(ctx))
+	exportKeys := s.cfg.GetBool("core.exportkeys")
```

Comment:
Shouldn't this also be a `s.cfg.GetM` since it could be mount-specific `exportkeys` setting?